test/releasers/node.ts

```diff
@@ -289,4 +289,31 @@ describe('Node', () => {
       req.done();
     });
   });
+
+  describe('coercePackagePrefix', () => {
```

Comment:
I think we need a test where `packageName` is not provided to `new Node({})` and that `releasePR.packagePrefix` is set correctly after `releasePR._run` is invoked since that's where the Node releaser [currently sets it](https://github.com/googleapis/release-please/pull/720/files?file-filters%5B%5D=.json&file-filters%5B%5D=.ts#diff-e6f703994fb6593e7ef82b0d3cdf990ebdd51764ac01a2723ffa45136d504730R38)

note: that might not be the best place for a releaser to set it but since it comes from inspecting source code, any usage of `this.packagePrefix` needs to wait till the releaser has had a chance to look it up dynamically.

Reply:
As long as we don't break the existing logic, I think it's shaken out both in this PR and in #725, that the fallback `packageName` lookup should be refactored a bit.

I wonder if we could pull it up to `releasea-pr.ts` instead, and have all releasers attempt the lookup., if it's supported by their language.

Reply:
actually, I think this is still broken for node as setting `this.packagePrefix` only happens in the constructor on this branch but `this.packageName` gets reset in `Node._run`...

Reply:
Yes, looks like this will cause the PR to be opened with the wrong packagePrefix